### PR TITLE
Update GFE VIP

### PR DESCRIPTION
**Pull request by adelez**

### CODE REVIEW

**Comment by yang-g** [comment]
Please merge with head.

File: tools/run_tests/run_interop_tests.py [hunk 1]
```diff
@@ -582,15 +582,15 @@ def aggregate_http2_results(stdout):
 # TODO(adelez): implement logic for errors_allowed where if the indicated tests
 # fail, they don't impact the overall test result.
 prod_servers = {
-    'default': ('grpc-test.sandbox.googleapis.com',
+    'default': ('172.217.3.228',
```
Comment (jtattermusch) [inline tools/run_tests/run_interop_tests.py]: why change the params for 'default'

**Comment by jtattermusch** [comment]
cloud_to_prod interop tests are failing.

**Comment by jtattermusch** [comment]
Please wait for interop tests to actually become green before applying the "ready to merge" label (interop tests finish in ~10 mins and for a PR that's touching interop test configuration there's no way I'd merge a change to interop tests without actually seeing a green test result).

**Comment by jtattermusch** [comment]
also, it doesn't look like this has been LGTMd (another reason why it's not mergeable).

**Comment by adelez** [comment]
Hmm, it's very strange that it fails. I'm changing the default because it seems GCE now connects to a CFE instead of GFE with the existing setup. So this PR is to use a particular VIP that's known to connect to GFE. I did manual test on the VM and it worked but I'm not sure why it fails on Jenkins. I'll keep investigating. By the way, yangg@ did say "merge with head". Is it not LGTM?

**Comment by yang-g** [comment]
I was asking you to update the code and resolve any conflict. Sorry I did not make it clearer.

I am a little concerned about maintaining these explicit IPs, but from what we chatted it seems like this is a temp thing for testing. So LGTM, please make sure the interop tests are green.

**Comment by adelez** [comment]
Actually I don't think this PR is necessary.

**Pull request closed without merging**